backend_jobs/ingestion_pipeline/pipeline_lib/database_functions.py

```diff
@@ -0,0 +1,143 @@
+"""
+  Copyright 2020 Google LLC
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+"""
+
+import random
+from datetime import datetime
+import apache_beam as beam
+import firebase_admin
+from firebase_admin import credentials
+from firebase_admin import firestore
+from google.cloud import firestore as cloud_firestore
+import geohash2
+
+IMAGES_COLLECTION=u'test3'
```

Comment:
probably a TODO for another CL, but these should come from some global config database script

Reply:
I have it in the Constance file.